### DISCUSSION TITLE
Can't install Windows 11 on M1 iPad Pro "This PC doesn't meet the minimum system requirements to install this version of Windows."

### DISCUSSION THREAD

**Discussion by MoltenMonster** [utmapp/UTM#4676] [UTM Help (iOS)] [answered]
**Describe the issue**
I tried setting up Windows 11 on my iPad Pro with 4GB of RAM but I got a message like this: 
![maxresdefault](https://user-images.githubusercontent.com/42647030/201939335-79bbb757-c352-49af-8524-b082c6e2c569.jpg)

**Configuration (required)**
* UTM Version: v4.0.9
* OS Version: iPadOS 15.6
* Device Model: M1 iPad Pro
* Is it jailbroken (name jailbreak used)? No. 
* How did you install UTM? Downloaded IPA from this repo and installed with AltStore.

**Upload VM**
[config.plist.zip](https://github.com/utmapp/UTM/files/10013320/config.plist.zip)
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by bzh32** [comment]
Using your iPad Pro will make your VM lack TPM 2. Try updating your UTM. UTM 4.0 works on mac.

**Comment by MoltenMonster** [comment]
Did u read the version? I installed 4.0.9

**Comment by MoltenMonster** [comment]
I installed the latest IPA in the releases page. And also, don't spam, this isn't a live chat.

**Comment by bzh32** [comment]
Did you choose the Install Windows 10 or higher option or download the ARM version from UUP dump?
Check the Minimum System Requirements for Windows 11: https://www.microsoft.com/en-us/windows/windows-11-specifications?r=1

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/guides/windows/
